static/test/javascripts/spec/common/analytics/interaction-tracking.spec.js

```diff
@@ -14,21 +14,14 @@ define([
                     trackSamePageLinkClick: sinon.spy(),
                     trackExternalLinkClick: sinon.spy()
                 })
-                .mock('common/modules/analytics/omniture', {
-                    go: sinon.spy(),
-                    trackSamePageLinkClick: sinon.spy(),
-                    trackExternalLinkClick: sinon.spy()
```

Comment:
@mkopka says that we rarely report on external page tracking. The comment left by Chris Birchell about re-impimenting the [500ms delay](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/analytics/interaction-tracking.js#L72) is untrue at the moment, as we have [explicitly turned this off in Omniture](https://github.com/guardian/frontend/blob/master/static/public/javascripts/vendor/omniture.js#L16). We will continue to report external link tracking in the same way as we do now in GA, so no data will be lost. I'm looking to implement this using the [Beacon API](https://developer.mozilla.org/en-US/docs/Web/API/Beacon_API) as a seperate PR to improve accuracy, and retain performance.